src/database/migrations/20240726043835-update-entity-types-validations-max-length.js

```diff
@@ -8,9 +8,12 @@ module.exports = {
 			title: { required: true, regex: '^[a-zA-Z0-9 <>_&-]{1,256}$' },
 			description: { required: true, regex: '^[a-zA-Z0-9 <>_&-]{1,2000}$' },
 			objective: { required: true, regex: '^[a-zA-Z0-9 <>_&-]{1,2000}$' },
-			name: { required: true, regex: '^[a-zA-Z0-9 <>_&-]{1,256}$' },
+			name: { required: true, regex: '^[a-zA-Z0-9 <>_&\\-,]{1,256}$' },
```

Comment:
@adithyadinesh0412  do we require // here ?

Reply:
@priyanka-TL yes, \\ is the escape character added because it was throwing error.